src/pages/recovery/recovery.js

```diff
@@ -68,7 +68,7 @@ const Recovery = () => {
             value={values.email}
             error={!!errorMsg}
           />
-          {errorMsg && <div className={styles.errorText}>{errorMsg}</div>}
+          <div className={styles.errorText}>{errorMsg || ''}</div>
```

Comment:
use helperText props for TextField component with alternative string with 1 whiteSpace

Reply:
Fixed.